src/pages/HealthContent.tsx

```diff
@@ -29,23 +29,26 @@ const HealthContent = () => {
   };
 
   return (
-    <div className={styles.healthContent}>
-      <SingleCarousel
-        cardName={styles.title}
-        title="Health Articles"
-        data={pageCards || []}
-      ></SingleCarousel>
-      <SingleCarousel
-        cardName={styles.title}
-        title="Health Tips"
-        data={pageCards || []}
-      ></SingleCarousel>
-      <SingleCarousel
-        cardName={styles.title}
-        title="Health News"
-        data={pageCards || []}
-      ></SingleCarousel>
-    </div>
+    <>
+      <div className={styles.healthContent}>
```

Comment:
Minor: This is hard to read. It could use some spacing, maybe comments to separate this out. And the components can self close, reducing some markup.

```js
  <div className={styles.healthContent}>

    {/* Health Articles */}
    <SingleCarousel
      cardName={styles.title}
      title="Health Articles"
      data={pageCards || []}
    />

    {/* Health Tips */}
    <SingleCarousel
      cardName={styles.title}
      title="Health Tips"
      data={pageCards || []}
    />

    {/* Health News*/}
    <SingleCarousel
      cardName={styles.title}
      title="Health News"
      data={pageCards || []}
    />

</div>
```